packages/f/fastcppcsvparser/xmake.lua

```diff
@@ -4,19 +4,18 @@ package("fastcppcsvparser")
     set_description("This is a small, easy-to-use and fast header-only library for reading comma separated value (CSV) files (by ben-strasser)")
 
     add_urls("https://github.com/ben-strasser/fast-cpp-csv-parser.git")
-    add_versions("0.0", "75600d0b77448e6c410893830df0aec1dbacf8e3")
+    add_versions(os.date("%Y.%m.%d"), "75600d0b77448e6c410893830df0aec1dbacf8e3")
```

Comment:
no, it should be commit(75600d0b) date in this git repo

75600d0b77448e6c410893830df0aec1dbacf8e3 -> 2021.01.03